createStore.js

```diff
@@ -14,8 +14,7 @@ function createStore(obj) {
         this.emit('change')
       }
     }
-  })
+  }, obj)
```

Comment:
I like the ability to override the initial state. I don't know about `setState` though.